tests/postgres_tests/test_integration.py

```diff
@@ -0,0 +1,16 @@
+import os
+import subprocess
+import sys
+
+from tests.postgres_tests import PostgreSQLSimpleTestCase
+
+
+class PostgresIntegrationTests(PostgreSQLSimpleTestCase):
+    def test_check(self):
+        os.chdir(os.path.dirname(__file__))
+        result = subprocess.run(
+            [sys.executable, '-m', 'django', 'check', '--settings', 'integration_settings'],
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+        )
+        self.assertEqual(result.returncode, 0, msg="django.contrib.postgres integration test failed")
```

Comment:
This isn't a helpful message -- tests show which test failed. What I meant was, if it's possible to display the check errors, that would be useful -- such as capturing stdout and/or stderror?